build.rs

```diff
@@ -1,16 +0,0 @@
-fn main() {
```

Comment:
The build script has been removed so the dependency on autocfg can also be removed.

Reply:
@taiki-e Fixed.